src/plugins/data/server/ui_settings.ts

```diff
@@ -715,5 +715,18 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
       }),
       schema: schema.boolean(),
     },
+    [UI_SETTINGS.SEARCH_TIMEOUT]: {
```

Comment:
Doesn't this call for a migration to copy old values into this setting?

Reply:
Just tested this, it doesn't need a migration since it's an advanced setting and the key doesn't change at all.

Reply:
Thanks, @lukasolson! That's also what I though 👍